en/power_module/hv_pm.md

```diff
@@ -0,0 +1,28 @@
+# HV PM(High voltage power module)
```

Comment:
Can you please name all power modules with prefix "CUAV" both here and in Summary? That just makes it clear that the product is CUAV from the outset.